tooling/build-types-and-manifests.ts

```diff
@@ -1,6 +1,7 @@
 /* ©️ 2016 - present FlowCrypt a.s. Limitations apply. Contact human@flowcrypt.com */
 import { readFileSync, writeFileSync } from 'fs';
 import { execSync as exec } from 'child_process';
+const MOCK_PORT = '[TEST_REPLACEABLE_MOCK_PORT]';
```

Comment:
We can't import the `MOCK_PORT` variable here as we do in other places (e.g., `import { MOCK_PORT } from './const.js';`) because this is a separate TypeScript project. 

To address this, I inserted `[TEST_REPLACEABLE_MOCK_PORT]` and replaced it later in `config-mock-build.sh`.

However, I feel that the current implementation has made the code a bit more complex. Let me know if this aligns with your expectations or if you have any other suggestions.

Reply:
@sosnovsky

Reply:
Yes, using separate `MOCK_PORT` variable adds some complexity, but it also removes a bit of confusion from generated original mock code where added `:8001` should be replaced later again.
Your implementation looks good, let's continue with this approach.
Thanks for asking!